src/libraries/System.Private.CoreLib/src/System/ThrowHelper.cs

```diff
@@ -556,6 +556,12 @@ internal static void ThrowApplicationException(int hr)
             throw ex;
         }
 
+        [DoesNotReturn]
+        internal static void ThrowFormatInvalidString()
```

Comment:
Use this in **ValueStringBuilder**?

Reply:
It's built into many libraries beyond Corelib.